stacks/DBStack.ts

```diff
@@ -18,11 +17,15 @@ export function DBStack(this: any, { stack }: StackContext) {
   });
 
   const myTable = new dynamodb.Table(this, 'Table', {
-    partitionKey: { name: 'MyPartitionKey', type: dynamodb.AttributeType.STRING },
+    partitionKey: {
+      name: 'MyPartitionKey',
+      type: dynamodb.AttributeType.STRING,
+    },
     sortKey: { name: 'MySortKey', type: dynamodb.AttributeType.STRING },
   });
 
   const uploads_bucket = new Bucket(stack, 'Uploads');
+  const speakingPollyBucket = new Bucket(stack, 'speakingPolly');
```

Comment:
Why not use the existing bucket?

```suggestion

  /* For now we;re using a bucket that is already filled
   * TODO: change this when the feature of adding questions is implemented
   */
  const speakingPollyBucket = s3.Bucket.fromBucketAttributes(
    this,
    'speakingPolly',
    {
      bucketArn: 'arn:aws:s3:::speaking-questions-polly',
    },
  );
  // const speakingPollyBucket = new Bucket(stack, 'speakingPolly');
```

use this import

```ts
import * as s3 from 'aws-cdk-lib/aws-s3';
```

Reply:
Not a bad idea, I will try that now